contrib/sbtc/token-key-admin/contracts/sbtc-token-key-admin.clar

```diff
@@ -0,0 +1,186 @@
+
+;; title: sbtc-token-key-admin
+;; version:
+;; summary:
+;; description:
+
+;; traits
+;;
+(impl-trait 'SP3FBR2AGK5H9QBDH3EEN6DF8EK8JY7RX8QJ5SVTE.sip-010-trait-ft-standard.sip-010-trait)
+
+;; token definitions
+;; 
+(define-fungible-token sbtc u21000000)
+
+;; constants
+;;
+(define-constant err-invalid-caller u1)
+(define-constant err-invalid-signer-id u2)
+(define-constant err-not-token-owner u3)
+
+;; data vars
+;;
+(define-data-var contract-owner principal tx-sender)
+(define-data-var coordinator (optional {addr: principal, key: (buff 33)}) none)
+(define-data-var num-keys uint u4000)
+(define-data-var num-parties uint u4000)
+(define-data-var threshold uint u2800)
+(define-data-var bitcoin-payout-address (optional (string-ascii 72)) none)
```

Comment:
Could we call this bitcoin-wallet-address? This address is used for receiving funds as well so payout could be confusing.

Reply:
Either one is fine with me, as long as there's a comment somewhere that explains it.

N.B. Treating this as a `(string-ascii 72)` is fine for now, since it doesn't appear to be used to interact directly with Bitcoin state.  In the future, this would almost certainly be represented as a PoX address, which has the type `{ version: (buff 1), hashbytes: (buff 32) }`.  But there's no need to do this for this PR.

Reply:
Done.